mcdb.spec

```diff
@@ -1,7 +1,7 @@
-### RPM external mcdb 1.0.2
+### RPM external mcdb 1.0.3
 Source: http://mcdb.cern.ch/distribution/api/%{n}-api-%{realversion}.tar.gz
 Requires: xerces-c
-Patch0: mcdb-1.0.2-gcc45
+#Patch0: mcdb-1.0.2-gcc45
```

Comment:
Yes nothing is using it in CMSSW we have not received any reply from generator L2 on cms-sw/cmssw#14728